### DISCUSSION TITLE
GameThread timed out waiting for RenderThread after 60.00 secs

### DISCUSSION THREAD

**Discussion by wujian20231101** [carla-simulator/carla#7076] [Q&A]
**./CarlaUE4.sh -RenderOffScreen**
4.26.2-0+++UE4+Release-4.26 522 0
Disabling core dumps.
LowLevelFatalError [File:Unknown] [Line: 1214]
GameThread timed out waiting for RenderThread after 60.00 secs
Signal 11 caught.
Malloc Size=65538 LargeMemoryPoolOffset=65554
CommonUnixCrashHandler: Signal=11
Malloc Size=131160 LargeMemoryPoolOffset=196744
Malloc Size=131160 LargeMemoryPoolOffset=327928
Engine crash handling finished; re-raising signal 11 for the default handler. Good bye.
Segmentation fault (core dumped)

**Comment by AuderySun** [comment]
@wujian20231101 Hello, Could you tell me if you've been able to solve this problem? I am meeting the same issue.

**Reply by tby-udel** [reply]
Hello, did you solve this problem?
